test/end-to-end/pages/thankyou/thankyou.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-const thankYouContent = require('app/resources/en/translation/thankyou');
+const thankYouContentEn = require('app/resources/en/translation/thankyou');
+const thankYouContentCy = require('app/resources/cy/translation/thankyou');
+const pageUnderTest = require('app/steps/ui/thankyou');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?